### PR TITLE
Hunter

### PR DESCRIPTION
**Pull request by Anarchid**
Fully scripted and kitted.

image

### CODE REVIEW

File: Objects3d/shiptorpraider.dae.lua [hunk 1]
```diff
@@ -0,0 +1,9 @@
+boat = {
+	radius = 50.00,
+	height = 35.00,
+	midpos = {0.00, 0.00, 0.00},
+	tex1 = "Hunter1.dds",
```
Comment (sprunk) [inline Objects3d/shiptorpraider.dae.lua:5]: would be good if the textures were called `shiptorpraider1.dds`

File: Objects3d/shiptorpraider.dae.lua [hunk 2]
```diff
@@ -0,0 +1,9 @@
+boat = {
```
Comment (sprunk) [inline Objects3d/shiptorpraider.dae.lua:1]:
```suggestion
return {
```

**Comment by sprunk** [comment]
is there underlying metal texture hidden by the teamcolor areas? it looks crayon.

File: Objects3d/shiptorpraider.dae.lua (hunk 1 above)
Comment (Anarchid) [reply Objects3d/shiptorpraider.dae.lua:5]: yes, but nothing else does it

File: Objects3d/shiptorpraider.dae.lua [hunk 3]
```diff
@@ -0,0 +1,9 @@
+return {
+	radius = 50.00,
+	height = 35.00,
+	midpos = {0.00, 0.00, 0.00},
+	tex1 = "Hunter1.dds",
+	tex2 = "Hunter2.dds",
+}
+
+return boat
```
Comment (sprunk) [inline Objects3d/shiptorpraider.dae.lua:9]:
```suggestion
```

**Comment by Anarchid** [comment]
> is there underlying metal texture hidden 100% by the teamcolor areas? it looks crayon.

All 3 texture layers have data under teamcolor, yes.

**Comment by Anarchid** [comment]
image

**Pull request merged**